application/src/test/java/org/opentripplanner/transit/speed_test/SpeedTest.java

```diff
@@ -144,7 +144,7 @@ public TestStatus status() {
 
   public static void main(String[] args) {
     try {
-      OtpStartupInfo.logInfo();
+      OtpStartupInfo.logInfo("Run SpeedTest");
```

Comment:
Why is this not written in all caps?

Reply:
I can do it. It does never appear in the same log as the others, so I thought it was less "intrusive" in small caps, where as it is ok to stand out from other log messages in the normal log. But, I guess we should stick to one way o formatting it. I will change it to use lower case and capital first letter.

Reply:
Renamed the others to match this one. The beginning of the log message is still in capital case, witch make these messages stand out from other log lines, but the "Run Server/Build ..." does not need to be in capitals.